images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -30,7 +30,8 @@ function Get-ClangVersions {
             $Matches.version
         }
     } | Sort-Object {[Version]$_}
-    return "Clang " + ($clangVersions -Join ", ")
+    $clangVersions = $clangVersions -Join ", "
+    return "Clang " + ($clangVersions -match '11.*' ? ($clangVersions + " (For Clang 11 you need to manually install required dependencies)") : $clangVersions)
```

Comment:
Actually, it looks like they need to be installed for all except the default one, like `libc++abi-9-dev` and `libc++-9-dev` for clang 9, so I'm not sure we need it in the readme. 🤔 @maxim-lobanov thoughts?

Reply:
May be we can don't write it in readme for now and see if we have more questions from customers. If it is unclear point- we can document it later

Reply:
@catthehacker remove it for now, please